addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2334,16 +2334,24 @@ export class OdooEditor extends EventTarget {
             // clean its children.
             for (const attribute of [...node.attributes]) {
                 // Keep allowed styles on nodes with allowed tags.
-                if (CLIPBOARD_WHITELISTS.styledTags.includes(node.nodeName) && attribute.name === 'style') {
-                    const spanInlineStyles = attribute.value.split(';').map(x => x.trim());
-                    const allowedSpanInlineStyles = spanInlineStyles.filter(rawStyle => {
-                        const [styleName, styleValue] = rawStyle.split(':');
-                        const style = CLIPBOARD_WHITELISTS.styles[styleName.trim()];
-                        return style && !style.defaultValues.includes(styleValue.trim());
-                    });
+                if (CLIPBOARD_WHITELISTS.styledTags.includes(node.nodeName) 
+                        && (attribute.name === 'style' || attribute.name in CLIPBOARD_WHITELISTS.styles)) {
```

Comment:
@dmo-odoo I add this condition because some style information coming from other software's tables is not inline.
To keep the background color I would have to keep style on `<td>` tags (that's where the style is added in our tables as well as LibreOffice tables). I could add it to the whitelist, but I wonder if we want to support all `CLIPBOARD_WHITELISTS.styles` on all `CLIPBOARD_WHITELISTS.styledTags` ?
Some of them may not be editable later from our editor, I guess that's what we want to prevent ?